.github/workflows/job-postcommit-java-nexmark-dataflowV2.yml

```diff
@@ -0,0 +1,151 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests for PostCommit Java Nexmark Dataflow V2 with Java 8, 11, and 17
+# using dataflow v2 runner
+
+name: PostCommit Java Nexmark Dataflow V2 - JAVA 8,11 and 17
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+permissions: read-all
+
+env:
+  BIGQUERYTABLE: nexmark
+  BIGQUERYDATASET: nexmark
+  PROJECT: apache-beam-testing
+  RESOURCENAMEMODE: QUERY_RUNNER_AND_MODE
+  EXPORTSUMMARYTOBIGQUERY: true
+  TEMPLOCATION: gs://temp-storage-for-perf-tests/nexmark
+  INFLUXDATABASE: beam_test_metrics
+  INFLUXHOST: http://10.128.0.96:8086 
+  BASEINFLUXMEASUREMENT: nexmark
+  EXPORTSUMMARYTOINFLUXDB: true
+  INFLUXRETENTIONPOLICY: forever
+  INFLUXTAGSJAVA8: '{"runnerVersion":"V2","javaVersion":"8"}'
+  INFLUXTAGSJAVA11: '{"runnerVersion":"V2","javaVersion":"11"}'
+  INFLUXTAGSJAVA17: '{"runnerVersion":"V2","javaVersion":"17"}'  
```

Comment:
The reason of having Influx tags in this way instead a matrix. is related a parse error using matrix and passing it into the gradlew command.